cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/analyzer/ProvisionResponse.java

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright 2021 LinkedIn Corp. Licensed under the BSD 2-Clause License (the "License"). See License in the project root for license information.
+ */
+
+package com.linkedin.kafka.cruisecontrol.analyzer;
+
+import static com.linkedin.cruisecontrol.common.utils.Utils.validateNotNull;
+
+
+/**
+ * Indicates the {@link ProvisionStatus} along with the recommended actions regarding the relevant status.
+ * Recommendations are only relevant to {@link ProvisionStatus#UNDER_PROVISIONED} and {@link ProvisionStatus#OVER_PROVISIONED}.
+ */
+public class ProvisionResponse {
+  private ProvisionStatus _status;
+  private final StringBuilder _recommendation;
+
+  /**
+   * Constructor to be used for provision statuses, for which the recommendations are relevant.
+   */
+  public ProvisionResponse(ProvisionStatus status, String recommendation) {
```

Comment:
Is recommendation going to be a human readable string or something else?  Can this be documented?